test/cpp/tensorexpr/test_simplify.cpp

```diff
@@ -1235,6 +1235,118 @@ TEST(Simplify, SimplifyDiv) {
   }
 }
 
+// NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
+TEST(Simplify, SimplifyDivWithLoopContext1) {
+  KernelScope kernel_scope;
+  // Stmt to simplify:
+  // for (int i = 0; i < 5; i++) {
+  //  A[i] = (i + 24) / 6;
+  //}
+  VarHandle i("i", kInt);
+  BufHandle a_buf("A", {5}, kInt);
+  auto forstmt = For::make(
+      i, 0, 5, Store::make(a_buf, {i}, (i + ExprHandle(24)) / ExprHandle(6)));
```

Comment:
Nit: I think you can write just "`(i + 24) / 6`", there is no need in wrapping int immediates with `ExprHandle` there. This applies to other tests too.